docs/user-guide/deployments/configuration.md

```diff
@@ -550,6 +550,20 @@ In the configuration files of standalone GreptimeDB, the value needs to be set a
 mode = "standalone"
 ```
 
+### Heartbeat configuration
```

Comment:
What components is this configuration used for? only frontend?  If it is, maybe we can move the content to PR #1262. If it isn't, please add the description about what components is this used for.

Reply:
Both datanode and frontend are using them.